cpp/test/Slice/errorDetection/TaggedParams.err

```diff
@@ -1,43 +1,39 @@
+TaggedParams.ice:15: value 4_294_967_296 for enumerator `e4' is outside the range of uint: [0..4_294_967_295]
```

Comment:
I don't personally find this underscore separation notation any easier to read than just forgoing separators...
Commas might be beneficial, but I don't think the underscores are.

Reply:
On the contrary, I think the `_` makes these numbers much more readable.

See https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-7.0/digit-separators (available in C# and apparently Java).

Reply:
It's available for specifying literals, but here we're printing it out as a string, so we're free to format it however we want, correct? I'd just personally prefer commas, but also it obviously doesn't matter very much :)